crates/cervo-runtime/src/runtime.rs

```diff
@@ -261,7 +262,7 @@ mod tests {
     fn test_run_for_rotation() {
         let mut runtime = Runtime::new();
         let mut keys = vec![];
-        for sleep in [0.02, 0.04, 0.06] {
+        for sleep in [0.02, 0.04, 0.06, 0.04] {
```

Comment:
Update the tests and asserts too to account for there being 4 items that should be in a specific order now.